cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -118,7 +120,13 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
     private val archive by option(
         "--archive",
         help = "Archive the downloaded source code as ZIP files to the output directory. Is ignored if " +
-                "'--project-url' is also specified."
+                "one of 'archive-all' and '--project-url' is also specified."
+    ).flag()
+
+    private val archiveAll by option(
+        "--archive-all",
+        help = "Archive the all the downloaded source code as a single zip file to the output directory. Is ignored " +
+                "if '--project-url' is also specified."
```

Comment:
"if one of '--archive' or '--project-url' is also specified."

Reply:
In the current logic, '--archive-all' takes the precedence over '--archive'

Reply:
Then I believe this should explicitly be mentioned also in this help text by inserting "Takes precedence over '--archive'." after the first sentence.